contrib/vdsm-dev

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env bash
```

Comment:
vdsm-dev is good name for your ~/bin directory, but as part of vdsm we don't need
the "vdsm-" prefix.

Is this tool meant to be used from the contrib directory or copied
to ~/bin?

Reply:
I have it in my bin folder, but it will work the same here.

If we get to the point where we want to integrate this, we can think of a better name. Maybe `dev-run` or something like that? Anyway, let's first see if this script has interest or nah.